### PR TITLE
Completed prep-javascript-arrays

### PR DESCRIPTION
**Pull request by PhilChoe**
image

### CODE REVIEW

File: prep-javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const firstElement = languages[0];
+console.log('The first element of the array is' + ' ' + firstElement);
+
+const thirdElement = languages[2];
+console.log('The third element of the array is' + ' ' + thirdElement);
+
+const length = languages.length;
+console.log('The length of the array is' + ' ' + length);
+
+const lastElement = languages[5];
```
Comment (robgmerrill) [inline prep-javascript-arrays/main.js]: Close! We want to use the length property
`languages[length - 1]`

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Close!

File: prep-javascript-arrays/main.js (hunk 1 above)
Comment (PhilChoe) [reply prep-javascript-arrays/main.js]: image

Comment (PhilChoe) [reply prep-javascript-arrays/main.js]: image

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**